### PR TITLE
Update contribute.rst according to request #980

### PR DESCRIPTION
**Pull request by bernhardlang**

### CODE REVIEW

File: docs/dev/contribute.rst [hunk 1]
```diff
@@ -42,7 +42,10 @@ Install PyMeasure in the editable mode.
     pip install -e .
 
 This will allow you to edit the files of PyMeasure and see the changes reflected. Make sure to reset your notebook kernel or Python console when doing so. Now you have your own copy of the development version of PyMeasure installed!
+Depending on your Python installation you may get an error messages saying that the file setup.py is missing or similar. Updating pip may solve the problem
 
+.. code-block:: bash
+    python -m pip install pip --upgrade
```
Comment (bmoneke) [inline docs/dev/contribute.rst:48]: I think, you need additional blank lines:
```suggestion
.. code-block:: bash

    python -m pip install pip --upgrade

```

**Comment by bmoneke** [comment]
I see, that this is your first contribution: You might add yourself to the authors.txt

**Comment by BenediktBurger** [comment]
Hey @bernhardlang 

I made a suggestion, how the code could work. Would you mind, merging that suggestion in your branch (possible via the github page)?

As a new contributor, you can add yourself to the authors.txt (in this pull request).

**Comment by BenediktBurger** [comment]
@bernhardlang , you can add yourself (in this pull request) to the `authors.txt` file.

This PR is ready to be merged. Please add yourself to the `authors.txt` or tell me, that you do not want to be named.

**Comment by BenediktBurger** [comment]
If you modify the file in the repository set to the main branch, it will create a new branch for each commited file.

You want to modify another file in your branch, so you have to _check out_ your branch `patch-1`, for example at the top of this page you can click on that link (or here: https://github.com/bernhardlang/pymeasure/tree/patch-1).
Now you can change a file and commit it to the branch `patch-1`.

Probably you have to merge the master branch first (if you have checked out your branch, you see the option to sync it), as there have been some more names added to the authors file in the mean time.

If it does not work as expected, I can merge your work and add you manually.

**Comment by bernhardlang** [comment]
Thanks, but there are still way too many missing bits for me. I'll leave it for today and ask the PyMoDAQ guys next week in Toulouse what to do in this specific case. That's probably going to be much easier in person.

**Comment by bernhardlang** [comment]
Hi Benedikt

At the end, there was no time in Toulouse to discuss this. So I'm back here. I added my name to AUTHORS.txt after having synced the fork. The file is still outdated with respect to the main branch which I guess is to be expected according to how git works. Meanwhile the test tells that there are conflicts in that file, as to be expected as well. Would there have been a more straight forward way?

bern

**Comment by BenediktBurger** [comment]
Thanks for having your name added.

I just resolved the conflict here on the github page with the "resolve conflict" button (you could have merged the master branch yourself and resolved the conflict, afterwards push the branch).

With this, your PR is ready to be merged. Thanks for your contribution and patience!

**Pull request merged**